src/test/web/step_definitions/loginSteps.ts

```diff
@@ -0,0 +1,47 @@
+import { Given, When, Then, setDefaultTimeout } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import { pageFixture } from '../../hooks/pageFixture';
+
+// import dotenv from 'dotenv';
+import dotenv from 'dotenv';
+ 
+setDefaultTimeout(60 * 1000 )
```

Comment:
A linter in the CI would be a great.
```suggestion
setDefaultTimeout(60 * 1000)
```